tools/dbtool.py

```diff
@@ -8,6 +8,7 @@
 import shutil
 import importlib
 import pathlib
+from dbtool_protect import *
```

Comment:
Is there a reason these dbtool changes are included in this PR?  Missed that from reviewing previously